src/components/OfferTile/Offer.scss

```diff
@@ -0,0 +1,45 @@
+.offer{
+  width: 1000px;
+  height: 250px;
+  margin: 16px auto;
+  border: 1px solid #eee;
+  box-shadow: 0px 2px 3px #ccc;
+  padding: 16px;
+
+  &:hover {
+    transition: 0.3s;
+    box-shadow: 2px 3px 6px 6px #ccc;
+  }
+
+  .offer-image-thumbnail {
+    display: block;
+    position: relative;
+    right: 5px;
+    bottom: 5px;
+    z-index: 1;
+  }
+
+  .offer-name {
+    display: block;
+    position: relative;
+    font-weight: 700;
+    width: 560px;
+    left: 25%;
+    bottom: 235px;
+  }
+
+  .offer-price {
+    display: block;
+    position: relative;
+    font-weight: 700;
+    width: 140px;
+    left: 86%;
```

Comment:
Why position it from the left? You could use `float: right`, or just use `right: 0`